src/content/eth/index.md

```diff
@@ -16,7 +16,9 @@ lang: en
 
 Ether, or ETH, is Ethereum's native currency. It is "digital money" that can be sent over the internet instantly and cheaply, and also be used in many [Ethereum-based applications](/en/dapps/).
 
-The easiest way to get ETH is to [buy some](https://docs.ethhub.io/using-ethereum/how-to-buy-ether/). There are many cryptocurrency exchanges that will allow you to buy ETH, but the one you should use will depend on where you live and how you want to pay.
+The easiest way to get ETH is to buy some from an exchange or [wallet](/wallets). Availability differs from country to country so check where you can buy ETH based on where you live first.
+
+<Button to="/get-eth">Find an exchange / wallet</Button>
```

Comment:
@ryancreatescopy general FYI - internal links should always have a trailing slash (i.e. `/get-eth/`)